web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -64,12 +66,12 @@ interface ProjectSize {
 }
 
 export interface ProjectStakeholder {
-  '@type': string;
+  '@type': ProfileType;
   'schema:name': string;
   'schema:description'?: string;
   'schema:image'?: string;
   'regen:adress'?: string;
-  'regen:showOnProjectPage': boolean;
+  'regen:showOnProjectPage': true;
```

Comment:
I want to double-check, that we want `true` here instead of `boolean`.

Reply:
good point, it will always be true for new projects for now, but it might actually be false in legacy projects... so probably best to keep it flexible and specify just a `boolean`